tensorflow/python/platform/ram_file_system_test.py

```diff
@@ -155,6 +155,24 @@ def foo(self):
     loaded = saved_model.load('ram://my_module')
     self.assertAllEqual(loaded.foo(), [1])
 
+  def test_join(self):
+    expected = 'ram://exists/a/b/c.txt'
+    self.assertEqual(
+      file_io.join('ram://', 'exists', 'a', 'b', 'c.txt'),
+      expected
+    )
+    self.assertEqual(
+      file_io.join('ram://exists', 'a', 'b', 'c.txt'),
+      expected
+    )
+    self.assertEqual(
+      file_io.join('ram://', 'exists/a', 'b', 'c.txt'),
+      expected
+    )
+    self.assertEqual(
+      file_io.join('ram://', 'exists', 'a', 'b/c.txt'),
+      expected
+    )
```

Comment:
Since I added `testJoinUrlLike` and `testJoinFilesystem` in `file_io_test.py`, should we minimize the changes by not adding this test (which would also mean not moving the test file)?

Reply:
Sounds good to me

Reply:
Cool, reverted those changes. Would you mind kicking off CI? Thanks